Tests.HealthCheck/appsettings.Development.json

```diff
@@ -0,0 +1,16 @@
+{
+  "HealthCheckEndpoints": {
+    "UserHealthCheckEndpoint": "http://localhost:9802/CBE144F5-AD53-4D0D-B6A1-39133E3F7D2D/hc",
+    "ProjectHealthCheckEndpoint": "http://localhost:9804/E68A3452-AEAF-4E28-92D5-CCC5EE9F0A4C/hc",
+    "TimeHealthCheckEndpoint": "http://localhost:9806/F784AA1B-28EC-483C-BA5B-634F72896A4A/hc",
+    "NewsHealthCheckEndpoint": "http://localhost:9808/930F4BC4-EE23-4E72-BB36-C0FF54AC887C/hc",
+    "MessageHealthCheckEndpoint": "http://localhost:9810/971E75B1-E475-4A2D-97A4-9A7FDE1FD4E9/hc",
+    "RightsHealthCheckEndpoint": "http://localhost:9812/8C8001A7-8424-4A73-8D80-14049760ACFE/hc",
+    "FileHealthCheckEndpoint": "http://localhost:9814/B59045FD-1267-4549-839A-2CD3376C7999/hc",
+    "CompanyHealthCheckEndpoint": "http://localhost:9816/47EC83DB-F697-4212-AA50-8D10EF1F17BC/hc",
+    "AuthHealthCheckEndpoint": "http://localhost:9818/6E921556-C11F-4DFB-909B-EDAF9BE2C589/hc"
+  },
+  "AuthSettings": {
+    "UriString": "http://localhost:9818/auth/login"
+  }
+}
```

Comment:
this can be reduce and write only url without guid. And why does not use https protocol?

Reply:
that get more detail hc from services, I think we should use hc with ID. 
I don't use https because on local machine it has no difference